src/Castle.Windsor/MicroKernel/Context/CreationContext.cs

```diff
@@ -126,7 +126,7 @@ private CreationContext()
 			resolutionStack = new Stack<ResolutionContext>(4);
 		}
 
-		public IDictionary AdditionalArguments
+		public Arguments AdditionalArguments
```

Comment:
Changing the type of `CreationContext.AdditionalArguments` should be noted in the changelog since it gets used in some extension points.

Reply:
Done.